gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
```

Comment:
Try running this locally:
```
python setup.py build_sphinx
open docs/_build/html/index.html
```
and look at the API docs for the `ObservationTableSummary` class.

You will find that it's not formatted correctly.
You have to remove the colon here and add an extra space below, i.e. make it look like this:
```
    Parameters
    ----------
    obs_table :  `~gammapy.data.ObservationTable`
        Observation table
    target_pos : `~astropy.coordinates.SkyCoord`
        Target position
```